Update administrativeunit.md (#11703)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/administrativeunit.md

|description|string|An optional description for the administrative unit.|
|displayName|string|Display name for the administrative unit.|
|id|string|Unique identifier for the administrative unit. Read-only.|

> |isManagementRestricted| Boolean| Indicates if the management rights on resources in the administrative units should be restricted to ONLY the administrators scoped on the AU object. If no value is specified, it will default to false.|

Reviewer: ```suggestion
|isManagementRestricted| Boolean| Indicates whether the management rights on resources in the administrative units should be restricted to only the administrators scoped on the AU object. Defaults to `false` If no value is specified.|
```
